manual/src/cmds/debugger.etex

```diff
@@ -79,6 +79,41 @@ file before giving control to the user. The default file is
 ".ocamldebug" in the current directory if it exists, otherwise
 ".ocamldebug" in the user's home directory.
 
+\subsection{ss:debugger-environ}{Environment Variables}
+
+The debugger supports the "BUILD_PATH_PREFIX_MAP" environment
+variable, as defined by the ``BUILD_PATH_PREFIX_MAP specification''
+(see
+\url{https://reproducible-builds.org/specs/build-path-prefix-map}).
+
+When the product being debugged was built, ``BUILD_PATH_PREFIX_MAP''
+was set by the build tools so that build-time absolute paths
+where mapped to abstract reproducible paths.
+
+Before ocamldebug is invoked, ``BUILD_PATH_PREFIX_MAP`` must again
+be set, but this time to cause the logical inverse of the
+first mapping. Typically that will be one by a build tool,
+but it may be done manually by a knowledgeable user.
```

Comment:
I would consider adding support in the compiler codebase for asking the BUILD_PATH_PREFIX_MAP reader (here `ocamldebug`) to invert the mapping themselves, to make life simpler for tools. (I think that @dbuenzli also had such a convenience feature in mind, but I am not sure.) But this could be done separately from the current PR.

Reply:
I will consider this. The mapping done during the build phase will involve the build directory, but during the deployment phase we also want to have the source directory and the opam directory as part of the map, so the deployment map cannot be purely derived from the build-phase map. I think it makes more sense for the build tool, e.g. Dune, to produce the inverse map than for the low-level tool.